Build/Build.cs

```diff
@@ -286,6 +285,8 @@ class Build : NukeBuild
                 customLogger: (_, msg) => Error(msg));
         });
 
+    string YarnCli => $"{ToolPathResolver.GetPackageExecutable("Yarn.MSBuild", "yarn.js", "1.22.19")} --silent";
```

Comment:
🤔 I actually liked the original place. Kept the "constants" like that at the top.

Reply:
Hmm.. I don't like scrolling around to see the implementation very much.. (in any project)

And because splitting up the `Build` class I have thought about to keep related things together like this

Reply:
Let's keep this at it's original place.
Then I don't have to scroll around when updating our dependencies.

Reply:
Ahh ok.. that'a fair point :)